common.sh

```diff
@@ -1,3 +1,6 @@
 SCRIPT_DIR="$(dirname "$(readlink -f "$BASH_SOURCE")")"
-export PATH="$SCRIPT_DIR/ext/k/k-distribution/bin:$PATH"
-export KOMPILE_OPTS="--backend llvm -ccopt $SCRIPT_DIR/time.cpp -ccopt $SCRIPT_DIR/tolowercase.cpp -ccopt -std=c++14 --hook-namespaces TIME --debug"
+PLUG_DIR="$SCRIPT_DIR/ext/blockchain-k-plugin"
+PLUG_SRC_DIR="$PLUG_DIR/plugin-c"
+BUILD_DIR="$PLUG_DIR/build"
+#export PATH="$SCRIPT_DIR/ext/k/k-distribution/bin:$PATH"
```

Comment:
Why is this commented out?
```suggestion
export PATH="$SCRIPT_DIR/ext/k/k-distribution/bin:$PATH"
```